tests/unit/filmtipset/test_filmtipset_ratings_parser.py

```diff
@@ -0,0 +1,100 @@
+        parser = FilmtipsetRatingsParser(None)
+        parser.movies = []
+        parser.site = site_mock
+        parser.site.site_name = 'Filmtipset'
+        parser.site.browser = browser_mock
+        parser.exports_folder = os.path.abspath(os.path.join(TESTDATA_PATH, 'exports'))
+        parser.downloaded_file_name = 'ft_betyg_2020-10-10.csv'
+        parser.csv_filename = '1234567890_filmtipset.csv'
+        parser.downloaded_file_name = os.path.join(os.pardir, 'filmtipset', 'ft_betyg_2020-10-10.csv')
+
+        parser.parse()
+
+        self.assertEqual(1, rename_csv_mock.call_count)
+        self.assertEqual(1, repair_mock.call_count)
+        self.assertEqual(1, parse_csv_mock.call_count)
+
+
+    @patch('RatS.filmtipset.filmtipset_ratings_parser.FilmtipsetRatingsParser._repair_line')
+    def test_repair_csv_file(self, line_mock):
+        original_test_file = os.path.join(TESTDATA_PATH, 'filmtipset', 'ft_betyg_2020-10-10.csv')
+        copied_file = original_test_file + '.cp'
+        copyfile(original_test_file, copied_file)
+        line_mock.return_value = 'a line\n'
+        FilmtipsetRatingsParser._repair_csv_file(copied_file) # pylint: disable=protected-access
+        self.assertEqual(8, line_mock.call_count)
+        os.remove(copied_file)
+
+    def test_repair_line(self):
+        tests = [
+            ('2012-02-05,Between You and Me;-1;2\n', '2012-02-05;"Between You and Me";-1;2\n'),
+            ('2004-01-16,Mina jag &amp; Irene;183505;3\n', '2004-01-16;"Mina jag &amp; Irene";183505;3\n'),
+            ('2004-12-08,Basic Instinct - iskallt begär;103772;3\n', '2004-12-08;"Basic Instinct - iskallt begär";103772;3\n'),
+            ('2019-04-27,Black Metal Satanica;;2\n', '2019-04-27;"Black Metal Satanica";;2\n'),
+            ('2019-04-27,A movie with "qoute";1234;2\n', '2019-04-27;"A movie with ""qoute""";1234;2\n'),
+            ('2020-09-22;"The King of Staten Island";9686708;4\n', '2020-09-22;"The King of Staten Island";9686708;4\n')
+        ]
+        for test in tests:
+            line, expected_line = test
+            newline = FilmtipsetRatingsParser._repair_line(line) # pylint: disable=protected-access
+            self.assertEqual(expected_line, newline)
+
+    @patch('RatS.filmtipset.filmtipset_ratings_parser.FilmtipsetRatingsParser._extract_imdb_informations')
+    @patch('RatS.utils.browser_handler.Firefox')
+    @patch('RatS.base.base_ratings_parser.RatingsParser.__init__')
+    @patch('RatS.filmtipset.filmtipset_ratings_parser.Filmtipset')
+    def test_convert_csv_row_to_movie(self, site_mock, base_init_mock, browser_mock, imdb_mock):
+        parser = FilmtipsetRatingsParser(None)
+        parser.movies = []
+        parser.site = site_mock
+        parser.site.site_name = 'Filmtipset'
+        parser.site.browser = browser_mock
+
+        expt_movie = {'title': 'Hunt for the Wilderpeople', 'filmtipset': {'my_rating': 10}}
```

Comment:
please don't use this kind of abbreviations, as it makes the code less readable.

```suggestion
        expected_movie = {'title': 'Hunt for the Wilderpeople', 'filmtipset': {'my_rating': 10}}
```